packages/core/src/models/extensions/dataSource.ts

```diff
@@ -6,28 +6,20 @@ import { ExtensionBase } from './base';
 import { VulcanExtension } from './decorators';
 
 // Original request parameters
-export interface RequestParameters {
-  [name: string]: any;
+export interface RequestParameter {
+  /** The index (starts from 1) of parameters, it's useful to generate parameter id like $1, $2 ...etc. */
+  parameterIndex: number;
```

Comment:
We can simply ignore this input argument and return the identifier whatever you want:
```ts
  public async prepare() {
    return `@${someRandom()}`
  }
```

I can't give you some meaningful parameter name because it might not be an input parameter, for example:
```sql
{% for val in someArray %}
{{ val }}  --- I don't know the name of value, only know the order (inde)
{% endfor %}
```

Reply:
Thanks @oscar60310 for replying my question and explain it!